src/resource_clients/key_value_store.js

```diff
@@ -1,5 +1,5 @@
 const ow = require('ow').default;
-const log = require('apify-shared/log');
+const log = require('@apify/log');
```

Comment:
In `index.js` it's imported from `default` and here it's imported directly 🤷

Reply:
good catch, apparently this is not tested

Reply:
Yeah, I don't think we're very thorough in testing loggers and logging 😞